appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -16,17 +17,19 @@ else{
   throw "Unrecognized repository provider: $repoProvider. Set the APPVEYOR_REPO_PROVIDER environment variable to a supported provider. Supported providers are: gitHub."
 }
 
-function Rebase-Uri($originalURI){
-  $replace = $originalURI.IndexOf("file:///")
-  if($replace -ne  -1){
-      $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString("file:///".Length))
-      $replaceLength = $caseSensitiveUri.IndexOf($projectSlug)
-      if($replaceLength -ne -1){
-          $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($replaceLength+$projectSlug.Length+1) -join '/')
-          return $builder.ToString()
-      }
+function Rebase-Uri($originalUri){
+  if($originalURI){
+    $fileUriPrefix = "file:///"
+    if($originalUri.StartsWith($fileUriPrefix)){
+        $caseSensitiveUri = Get-CaseSensitivePath $originalUri.SubString($fileUriPrefix.Length 
+        $projectSlugIndex = $caseSensitiveUri.IndexOf($projectSlug)
+        if($projectSlugIndex -ne -1){
+            $builder.Path = ($repoName, $repoCommit, $caseSensitiveUri.SubString($projectSlugIndex+$projectSlug.Length+1) -join '/')
+            return $builder.ToString()
+        }
+    }
   }
-  Write-Error "Unable to rewrite URI"
+  Write-Host "Unable to rewrite URI: " $originalURI
```

Comment:
$originalUri. Also on the line below. PS is case-insensitive for variable names, which is why you have to watch out for this yourself.